docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md

```diff
@@ -0,0 +1,88 @@
+# Identifiers
+
+## Overview
+By default LORIS provides 3 different identifiers for each candidate: 
+
+- **CandID**, also known as the **DCCID**, is a unique randomized 6-digit numeric ID (e.g. '436792') assigned automatically by LORIS upon the candidate's registration. The CandID is not configurable.
+- **PSCID** (Project Study Center ID) is a unique configurable ID. It can be set up to be either manually entered when registering a candidate or automatically generated and usually contains the site or project abbreviation followed by sequential or randomized characters (e.g. 'MTL0006'), but its exact format is customizable.
+- **ExternalID** is a unique configurable ID. It can be set up to be either manually entered when registering a candidate or automatically generated and it is generally completely de-identified (no site or project information incorporated) to be used for data dissemination.
+
+The format and the generation of the **PSCID** and **ExternalID** must be configured by an admin by editing the `config.xml` file.
+
+## Configuration
+
+### Configuration from the front end
+_not yet available_
+
+### Configuration from SQL
+_not yet available_
+
+### Configuration from the config.xml file
+
+Both the format and the generation of PSCIDs can be configured by an administrator in the `config.xml` file of the `%LORIS_ROOT%/project/` directory. these settings are applied to any and all new candidates.
```

Comment:
```suggestion
Both the format and the generation of PSCIDs can be configured by an administrator in the `config.xml`. These settings are applied to any and all new candidates.
```

You're assuming the location of config.xml. There are many places it could be located.

Reply:
@driusan I think your suggestion should read 

"in the `config.xml` file. These "

or "in `config.xml`. These"

Reply:
thanks, fixed the suggestion.

Reply:
you are correct